release notes/v0.37.0.md

```diff
@@ -0,0 +1,73 @@
+k6 v0.37.0 is here! 🎉 Mainly it contains fixes and ongoing efforts with refactoring.
+
+## New Features!
+
+### Added an option to output k6 logs to a file through --log-output ([#2285](https://github.com/grafana/k6/pull/2285))
+
+This is on top of the supported stdout/stderr and loki instance.
+
+The feature speaks for itself with examples of usage:
+
+```sh
+k6 run --log-output file=./k6.log --logformat json ./examples/stages.js
+```
+
+with a defined minimal log level:
+
+```sh
+k6 run --log-output file=./k6.log,level=info --logformat json ./examples/stages.js
+```
+
+Thanks, @[alyakimenko](https://github.com/alyakimenko) for the contribution!
+
+**Docs**: [Using file output](https://k6.io/docs/using-k6/options/#file)
+
+### Added experimental basic event loops ([#882](https://github.com/grafana/k6/issues/882))
+
+We added basic event loop functionality ([#2228](https://github.com/grafana/k6/pull/2228) and [#2373](https://github.com/grafana/k6/pull/2373)). This is a first step and currently isn't used by anything, but is now available to extensions. In following releases this will be used by k6 and will mean that some APIs will start having callbacks or return Promises. But first it's likely that it will be used in some experimental extensions.
+
+If you are an extension developer please use it and give your feedback. But take into consideration that it's likely that the API may change
+
+## Extensions
+
+`v0.37.0` finalizes ([#2376](https://github.com/grafana/k6/pull/2376) and [#2384](https://github.com/grafana/k6/pull/2384)) the switching of our internal modules to [a new Go/JavaScript module API](https://k6.io/docs/extensions/guides/create-an-extension/#advanced-javascript-extension). As the last internal the GRPC module was refactored ([#2365](https://github.com/grafana/k6/pull/2365)).
+
+It's important to highlight that the old API is deprecated and will be removed in the next k6's release (`v0.38.0`). Every extension that isn't using that API will get a warning message, like:
+
+```sh
+WARN[0000] Module 'k6/x/sql' is using deprecated APIs that will be removed in k6 v0.38.0, for more details on how to update it see https://k6.io/docs/extensions/guides/create-an-extension/#advanced-javascript-extension
+```
+
+We did migrations for some xk6 extensions (see connected issues to the task [#2344](https://github.com/grafana/k6/issues/2344)). The pull requests can serve as examples on how to transition your extension to the new API.
+
+## Docker Repository
+
+We migrated our docker hub's repository from `loadimpact/k6` to the [grafana/k6](https://hub.docker.com/r/grafana/k6) ([#2377](https://github.com/grafana/k6/pull/2377)).
+
+```sh
+docker run -i grafana/k6 run - <script.js
```

Comment:
Should we change the `Dockerfile` to use the following?
```shell
ENTRYPOINT ["k6", "run"]
```
This would make the secondary "run" unnecessary...I feel like anyone running the Docker image would be doing local testing. Also...my personal preference is to always include the `--rm` to reduce Docker bloat.
```shell
docker run -i grafana/k6 --rm -<script.js
```

Reply:
Agreed on suggesting `--rm`, but not sure about changing the entrypoint. The current way allows using other popular commands (`cloud`, `archive`, `version`, etc.), some of which may be cumbersome with Docker, but it would avoid having to specify `--entrypoint` every time. Plus I like that the same command after `docker run -i grafana/` can just be copy/pasted to work outside of Docker. :slightly_smiling_face:

Reply:
I agree that maybe it's good to change the command, but I believe that not here, but [in the docs](https://github.com/grafana/k6-docs/blob/main/src/data/markdown/translated-guides/en/01%20Getting%20started/03%20Running%20k6.md?plain=1#L145) :smile: 

This place emphasizes the new repository, not the working example (see the script), so shorter is better. What do you think?

Reply:
https://github.com/grafana/k6-docs/pull/589